docs/install/install-eventing-with-yaml.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Installing Knative Eventing using YAML files"
+linkTitle: "Install Eventing using YAML"
+weight: 03
+type: "docs"
+showlandingtoc: "false"
+---
+
+This topic describes how to install Knative Eventing.
+You can install the Knative Eventing component by applying YAML files using the `kubectl` CLI.
+
+When installing Knative, you can install the Serving component, Eventing component, or both on your cluster.
+For how to install the Serving component, see [Installing Serving using YAML files](./install-serving-with-yaml.md)
+
+## Prerequisites
+
+Before installation, you must meet the prerequisites.
+See [Knative Prerequisites](./prerequisites.md).
+
```

Comment:
We discussed this in the Docs WG meeting. Our compromise was to have the Prereqs in another topic and provide a link.

Reply:
The idea being that we wouldn't have to update things everywhere. 

In the future, we could have this appear as a "Toggle" block which references a single source of truth for Pre-Reqs...

Reply:
Another reason I hate markdown - no content re-use / inclusion 🙃 Do you know if mkdocs adds any functionality to provide that?
If we used asciidoc we would just have one content module that we `include::` in different places, just sayin' 😂 

Fair enough if we're just linking to it for now, but as a reader I just think it would annoy me having to navigate away to another page to read prereqs and then come back here to do the actual install.